wire-ios/Wire-iOS/Sources/UserInterface/UserProfile/Devices/UserClientCell.swift

```diff
@@ -88,7 +88,7 @@ final class UserClientCell: SeparatorCollectionViewCell {
         addBorder(for: .bottom)
         accessoryIconView.translatesAutoresizingMaskIntoConstraints = false
         accessoryIconView.contentMode = .center
-        accessoryIconView.image = .init(resource: .rightChevron).withRenderingMode(.alwaysTemplate)
```

Comment:
This needs to stay here so the color changes the right way between light and dark mode.

Reply:
Since the rendering mode was set each time this image asset is used, I set the rendering mode in the asset catalog instead and removed it from code.
The end result should be the same, please correct me if I'm wrong.

Reply:
Ok I missed that then it's good to go! 🎉